code/game/gamemodes/cm_initialize.dm

```diff
@@ -406,7 +396,7 @@ datum/game_mode/proc/initialize_special_clamps()
 		return pick(available_xenos_non_ssd) //Just picks something at random.
 
 	new_xeno = input("Available Xenomorphs") as null|anything in available_xenos
-	if (!istype(new_xeno) || !xeno_candidate)
+	if (!istype(new_xeno) || !xeno_candidate || !xeno_candidate.client)
```

Comment:
512 you can further combine things like this and it improves performance very slightly to use
`if(!xeno_candidate?.client)`

Reply:
Very useful. Lots of instances that could use this kind of thing. I'll start using it from now on.